en/docs/administer/product-level-security-guidelines.md

```diff
@@ -268,3 +268,54 @@ Log forging can be prevented by appending a UUID to the log message.
 If mutual SSL is enabled, [enable intermediate certificate validation](../../develop/authenticating-and-authorizing-rest-apis/#configure-intermediate-certificate-validation) as well to make sure that only certificates signed by the issuers mentioned in the **IntermediateCertValidation** configuration are allowed to be used during mutual SSL authentication.
 
 If mutual SSL authentication capabilities are not required, you can [disable it](../../administer/enabling-mutual-ssl/#enabling-mutual-ssl-in-the-wso2-is).
+
+## Configuring client authentication
+
+Client authentication is used to identify the application or the client that is making the request. 
+The web applications provided out-of-the-box use a set of default credentials to authenticate with WSO2 Identity Server REST APIs that are marked as **secure** under the 'ResourceAccessControl' tag of the the`<IS_HOME>/repository/conf/identity/identity.xml` file. 
+
+Follow the steps below to change the default credentials.
+
+1.  Before applying the configurations, make sure that you get the latest WUM updates for this release. See [WSO2 
+Update Manager](https://wso2.com/updates/wum)(WUM) to get any
+    fixes or latest updates for this release.
+
+    !!! note "Important"
+        If you are upgrading to use this version in your production
+        environment, use WSO2 Update Manager to get the latest updates
+        available for WSO2 IS 5.9.0. For more information on how to use WSO2
+        Update Manager, see [Updating WSO2 Products](https://docs.wso2.com/display/updates/Using+WSO2+Update+Manager).        
+2.  Shut the server down in case you have already started it. 
+3.  Add the following configuration changes to the `<IS_HOME>/repository/conf/deployment.toml` file.
+    
+    - Add the `app_password` property and enter a preferred password as the value.
+      
+       ``` toml
+       [identity.auth_framework.endpoint] 
+       app_password="<value of preferred password>"
+
+        ```  
+    - Add the `hash` property and enter the SHA-256 hash value of the `app_password` as the property value.
+    
+         ``` toml
+         [account_recovery.endpoint.auth]
```

Comment:
Are we only explaining the account_recovery.endpoint.auth.hash? what about account_recovery.endpoint.auth.name?

Reply:
We can omit it as there is no security requirement for changing the name.